bakery_service/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM openjdk:21-jdk
+
+COPY build/libs/*.jar /bakery/bakery_service-0.0.1-SNAPSHOT.jar
+
+WORKDIR /bakery
+
+ENV DATABASE_HOST localhost
+ENV DATABASE_PORT 5555
+ENV DATABASE_USERNAME x
+ENV DATABASE_PASSWORD x
+ENV DATABASE_NAME bakerydb
+
+ENV SPRING_PROFILES_ACTIVE=docker
+
+EXPOSE 8080
```

Comment:
if you want to run this Dockerfile as a way to `docker build & run` the app then we don't  need to have this `expose` here. Rather we need `-p 8080:8080` flag upon running `docker run` command. The difference is this `EXPOSE` will expose port in the private docker network (so for example when we would want to connect with another service running on docker then this expose would tell the other service which port to use). However, if we just want to run `docker build` and then `docker run` from the terminal then we don't need this here.
More here https://www.reddit.com/r/docker/comments/u9ooj5/please_explain_to_me_what_is_the_difference/